egcg_core/archive_management.py

```diff
@@ -83,13 +85,18 @@ def release_file_from_lustre(file_path):
         return True
 
 
-def register_for_archiving(file_path):
+def register_for_archiving(file_path, strict=False):
     if is_register_for_archiving(file_path):
         return True
     cmd = 'lfs hsm_archive %s' % file_path
     val = _get_stdout(cmd)
     if val is None or not is_register_for_archiving(file_path):
-        raise ArchivingError('Registering %s for archiving to tape failed' % file_path)
+        if strict:
+            raise ArchivingError('Registering %s for archiving to tape failed' % file_path)
+        # Registering for archive can sometime take time so give it a second
+        sleep(1)
+        return register_for_archiving(filter, strict=True)
```

Comment:
If something external like `lfs` breaks, this would recur indefinitely. Should we have a retry limit like in `notifications.email` before raising an error?